rnnalyse/classifiers/dc_trainer.py

```diff
@@ -65,8 +66,16 @@ def train(self, train_subset_size: int = -1, train_test_split: float = 0.9) -> N
         for a_name in self.activation_names:
             data_dict = self.activations_reader.create_data_split(a_name,
                                                                   train_subset_size,
+
                                                                   train_test_split)
 
+            # Calculate class weights
```

Comment:
Great! Should we maybe provide the option to overwrite this in case someone doesn't want to use class weights (for whatever reason)?

Reply:
Done